src/providers/preview/graphicspreview.ts

```diff
@@ -26,6 +26,11 @@ export class GraphicsPreview {
     }
 
     async provideHover(document: vscode.TextDocument, position: vscode.Position): Promise<vscode.Hover | undefined> {
+        const configure =vscode.workspace.getConfiguration('latex-workshop')
+        const preview = configure.get('intellisense.includegraphics.preview.enabled') as boolean
+        if (!preview) {
+            return undefined
+        }
```

Comment:
Based on the above comment, `intellisense.includegraphics.preview.enabled` is not the correct configuration variable to be used here.